src/System.Security.Cryptography.Cng/src/System/Security/Cryptography/ECDsaCng.Key.cs

```diff
@@ -50,6 +40,39 @@ private set
                 KeySize = key.KeySize;
             }
         }
+
+        public CngKey GenerateNewKey()
```

Comment:
We discussed the capability to generate a new key (without having to re-import or create a new cng\openssl) but I don't recall the parameters. I was assuming they would use the existing curve (or keysize for the older code). Perhaps we should take this offline.

That method was added before I started on openssl, before I realized that openssl doesn't even have a key class. So adding it to ECDsa makes sense for both, and perhaps not even on CngKey